src/panels/config/info/system-health-card.ts

```diff
@@ -104,6 +125,25 @@ class SystemHealthCard extends LitElement {
     }
   }
 
+  private _copyInfo(): void {
+    const copyElement = this.shadowRoot?.querySelector(
+      ".table-homeassistant"
+    ) as HTMLElement;
+
+    const selection = window.getSelection()!;
+    const range = document.createRange();
+
+    range.selectNodeContents(copyElement);
+    selection.removeAllRanges();
+    selection.addRange(range);
+
+    document.execCommand("copy");
+    window.getSelection()!.removeAllRanges();
+
+    this._toolTip!.show();
+    setTimeout(() => this._toolTip?.hide(), 3000);
```

Comment:
`_toolTip!`, `this._toolTip?`. Are we sure it is present or not? 😜

Reply:
@bramkragten I think this is copied from my code on the log book. The problem for this comes if the page is changed or the tooltip is no longer there then an error would occur. So this is correct

Reply:
true